README.md

```diff
@@ -42,46 +42,176 @@ These elements are described in different folders. All the modelling happens wit
+
+Supported platforms:
+- GNU/Linux distributions (in particular Debian and Ubuntu);
+- Mac OS X;
+- Microsoft Windows (we recommend using [ConEmu](https://conemu.github.io/) instead of the default console).
+
+Other OS should work if they can execute Python and NumPy.
+
+### Setting-up a Virtual Environment with Pew
+
+We recommend using a [virtual environment](https://virtualenv.pypa.io/en/stable/) (abbreviated as "virtualenv") with a virtualenv manager such as [pew](https://github.com/berdario/pew).
+
+- A [virtualenv](https://virtualenv.pypa.io/en/stable/) is a project specific environment created to suit the needs of the project you are working on.
+- A virtualenv manager, such as [pew](https://github.com/berdario/pew), lets you easily create, remove and toggle between several virtualenvs.
+
+To install pew, launch a terminal on your computer and follow these instructions:
+
+```sh
+python -V # You should have python 2.7.9 or better installed on your computer.
+# If not, visit http://www.python.org to install it and install pip as well.
 ```
-pip install -e ".[test]"
+
+```sh
+pip install --upgrade pip
+pip install pew  # if asked, answer "Y" to the question about modifying your shell config file.
+```
+To set-up and create a new a virtualenv named **openfisca** running python2.7:
+
+```sh
+pew new openfisca --python=python2.7
+```
+
+The virtualenv you just created will be automatically activated. This means you will operate in the virtualenv immediately. You should see a prompt ressembling this (depending on the OS you are using) :
+```sh
+Installing setuptools, pip, wheel...done.
+Launching subshell in virtual environment. Type 'exit' or 'Ctrl+D' to return.
+openfiscabash-3.2$ 
+```
+Additional information :
+- Exit the virtualenv with `exit` (or Ctrl-D).
+- Re-enter with `pew workon openfisca`.
+
+:tada: You are now ready to install this OpenFisca Country Package!
+
+We offer 2 install procedures. Pick procedure A or B below depending on how you plan to use this Country Package. 
+
+### A. Minimal Installation (Pip Install)
+
+Follow this installation if you wish to:
+- run calculations on a large population;
+- create tax & benefits simulations;
+- write an extension to this legislation (e.g. city specific tax & benefits);
+- serve your Country Package with the OpenFisca Web API.
+
+For more advanced uses, head to the [Advanced Installation](#advanced-installation-git-clone).
+
+#### Install this Country Package with Pip Install
+
+Inside your virtualenv, check the prerequisites:
+
+```sh
+python --version  # should print "Python 2.7.xx".
+#if not, make sure you pass the python version as an argument when creating your virtualenv
+```
+
+```sh
+pip --version  # should print at least 9.0.
+#if not, run "pip install --upgrade pip"
+```
+Install the Country Package:
+
+```sh
+#Example: Openfisca-France
+pip install openfisca-france
```

Comment:
I'd let `openfisca_country_template`.

This way when the country package builder runs the script detailed in the "Bootstrapping" section, `openfisca_country_template` will be automatically renamed to the right country package name.